### ISSUE TITLE
Blobity bugging out navbar

### ISSUE THREAD

**Issue by GabrielF11** [GabrielF11/gabrielf11.ga#17]
I'm not sure how or when this happened, but now Blobity is showing behind some elements like the navbar and cards on the project page.

https://user-images.githubusercontent.com/88408682/141878769-8c2931b0-f4ff-4ea8-ad09-c861beec9b8c.mp4

**Comment by Turtle84375** [comment]
I don't actually see any problem on my end. I use ChromeOS on Chromebook (ikr), maybe it's just something with MacOS?

Here's what I see:

https://user-images.githubusercontent.com/91347263/142295796-a4793119-264a-45d5-8c00-c15740c19d26.mp4

(oof sorry if it's laggy, that's Chromebook for you...)

The grey blob goes behind elements, is that the issue you wanted to fix? Personally, I don't think it's really an issue, because if that blob went in front of things like the navbar, you wouldn't be able to read the words :P

In other words, I think it's meant to go behind those elements, or if it's not, I think it looks better going behind anyway. Now, what is _shouldn't_ be doing, which I saw in your video, is bugging out, which didn't happen to me, so it could be something to do with OS.

**Comment by GabrielF11** [comment]
Ok so I just check and it's fine now...  that was weird. Must've been a Blobity change, that was then undone...? Resolved.
